package.json

```diff
@@ -103,12 +103,14 @@
     "defs": "node scripts/export-service-definitions-cli.js > service-definitions.yml",
     "build": "run-s defs features && gatsby build",
     "heroku-postbuild": "run-s --silent build",
-    "start:server": "cross-env NODE_CONFIG_ENV=development node server 8080",
-    "now-start": "node server",
+    "start:server:prod": "node server",
+    "now-start": "npm run start:server:prod",
+    "start:server:e2e-on-build": "node server 8080",
+    "start:server": "cross-env NODE_CONFIG_ENV=development nodemon server 8080",
     "prestart": "run-s --silent depcheck defs features",
     "start": "concurrently --names server,frontend \"npm run start:server\" \"cross-env GATSBY_BASE_URL=http://localhost:8080 gatsby develop --port 3000\"",
     "e2e": "start-server-and-test start http://localhost:3000 test:e2e",
```

Comment:
In `npm run e2e` I got:
```
[server] [nodemon] ignoring: package.json **/*.spec.js **/*.tester.js **/*.integration.js ./frontend/**/* ./public/**/*
[server] [nodemon] watching: *.*
[server] [nodemon] watching extensions: js
...
  Main page
[server] [nodemon] files triggering change check: cypress/videos/main-page.spec.js.mp4
[server] [nodemon] matched rule: **/*.*
[server] [nodemon] changes after filters (before/after): 1/0
[server] [nodemon] files triggering change check: cypress/videos/main-page.spec.js.mp4
[server] [nodemon] matched rule: **/*.*
[server] [nodemon] changes after filters (before/after): 1/0
```
So I added `"**/*.js.*",` to `nodemonConfig.ignore`
In another run I got:
```
  - Started processing:   Compressing to 32 CRF
[server] [nodemon] files triggering change check: cypress/videos/main-page.spec.js-compressed.mp4
[server] [nodemon] matched rule: **/*.*
[server] [nodemon] changes after filters (before/after): 1/0
[server] [nodemon] files triggering change check: cypress/videos/main-page.spec.js-compressed.mp4
[server] [nodemon] matched rule: **/*.*
[server] [nodemon] changes after filters (before/after): 1/0
[server] [nodemon] files triggering change check: cypress/videos/main-page.spec.js-compressed.mp4
[server] [nodemon] matched rule: **/*.*
```
So I added `"**/*.js-*.*",` and after this I have no info about change check. Changes in mp4 does not restart server but it's strange for me that `main-page.spec.js.mp4` and `main-page.spec.js-compressed.mp4` trigger checks.

Reply:
Would ignoring `cypress/` solve this?

Reply:
Sounds good to me :-)